README.md

```diff
@@ -1,5 +1,37 @@
 # Praat: doing phonetics by computer
 
+Welcome to Praat! Praat is a speech analysis tool used for doing phonetics by computer. Praat can analyse, synthesize, and manipulate speech, and create high-quality pictures for analysis. Praat allows you to automatically apply filters based on theories of speech production to your speech recordings, so that some of the analytical work is taken off of you. Some of Praat’s most prominent features are:
```

Comment:
There's a vague tendency to keep lines in the README down to around 80-100 characters for people with old-school text editors. Perhaps you can chop this paragraph up a little with some newlines.

Reply:
"analyse" (British) and "synthesize" (US-ish) seems a bit grating in one line. I don't know if Praat has an official (spelling) language for documentation. "synthesise" is an acceptable British spelling, or "analyze" to switch to US.

Reply:
Perhaps "burden" instead of "work"? The phrase "work is taken off of you" sounds weird to me. Maybe "work is done for you"?

Reply:
**Somewhere** there's an Oxford comma there but I can't decide which one (there are two ", and" bits in this paragraph).